Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -135,6 +135,7 @@ namespace AzToolsFramework
             * Gets the ids for the entities in the Instance DOM.  Can recursively trace all nested instances.
             */
             void GetNestedEntityIds(const AZStd::function<bool(AZ::EntityId)>& callback) const;
+            size_t GetNestedEntityAliasCount() const;
```

Comment:
nit: Not clear to me what the difference is between GetEntityAliasCount and GetNestedEntityAliasCount, I would consider adding public comments to clarify difference

Reply:
Removed entirely as it's not needed